assessment/report_generator.go

```diff
@@ -45,15 +45,10 @@ type SchemaReportRow struct {
 	codeSnippets      string
 }
 
-func generateAndDumpSchemaReport(dbName string, assessmentOutput utils.AssessmentOutput) {
-	//pull data from assessment output
-	//Write to report in require format
-	//publish report locally/on GCS
-	//logger.Log.Info(fmt.Sprintf("%+v", assessmentOutput))
-
-	f, err := os.Create(dbName + "_schema.txt")
+func dumpCsvReport(fileName string, records [][]string) {
+	f, err := os.Create(fileName)
 	if err != nil {
-		logger.Log.Error(fmt.Sprintf("Can't create schema file %s: %v", dbName, err))
+		logger.Log.Error(fmt.Sprintf("Can't create schema file %s: %v", fileName, err))
```

Comment:
nit: this method is called in the schema and non schema flow. this should  just be csv file or something.

Reply:
Done